bazel_tools/haskell-keep-hs-extra-libs.patch

```diff
@@ -0,0 +1,25 @@
+rules_haskell excludes Haskell libraries from the extra-libraries field in
+package config files to avoid linker errors in a Haskell-CC-Haskell sandwhich.
+See https://github.com/tweag/rules_haskell/issues/873.
+Here we carry Haskell libraries only through the extra-libraries field, so we
+have to undo those patches.
+diff --git a/haskell/private/actions/link.bzl b/haskell/private/actions/link.bzl
+index cb6aa19..65cd2c6 100644
+--- a/haskell/private/actions/link.bzl
++++ b/haskell/private/actions/link.bzl
+@@ -485,15 +485,6 @@ def _add_external_libraries(args, ext_libs):
+     deduped = list.dedup_on(get_lib_name, ext_libs)
+ 
+     for lib in deduped:
+-        # This test is a hack. When a CC library has a Haskell library
```

Comment:
Not sure if the fact that we now have patches to remove hacks in rules_haskell is a good or a bad thing :smile: